DFC.ServiceTaxonomy.Events/Activities/Tasks/PublishToEventGridTask.cs

```diff
@@ -203,10 +214,40 @@ public override IEnumerable<Outcome> GetPossibleOutcomes(WorkflowExecutionContex
             }
         }
 
-        private async Task PublishContentEvent(WorkflowExecutionContext workflowContext, ContentItem contentItem, string eventType)
+        private async Task PublishContentEvents(
+            WorkflowExecutionContext workflowContext,
+            ContentItem contentItem,
+            ContentEventType eventType)
+        {
+            switch (eventType)
+            {
+                case ContentEventType.Published:
+                case ContentEventType.Unpublished:
+                    await PublishContentEvent(workflowContext, contentItem, _publishedContentItemVersion, eventType);
+                    //todo: if there wasn't a draft version, would have to publish unpublished with pewview graph too
```

Comment:
Typo in the todo, not that it matters much :)

Reply:
i'll fix it in my current branch 😄